core/templates/pages/donate-page/donate-page.component.spec.ts

```diff
@@ -17,6 +17,7 @@
  */
 
 import { TestBed } from '@angular/core/testing';
+import { EventEmitter } from '@angular/core';
```

Comment:
Please separate the library imports by an empty line.

Reply:
dittoelsewhere

Reply:
Done!